example/spec/s3Access/s3AccessSpec.js

```diff
@@ -45,38 +81,76 @@ describe('When accessing a bucket directly', () => {
   });
 
   describe('with credentials associated with an Earthdata Login ID', () => {
-    let credentials;
-    let s3EdlCreds;
+    let creds;
 
     beforeAll(async () => {
-      /**
-       * TO DO:
-       * Fetch the credentials from the endpoint
-       */
-      // credentials = .....
-
-      // s3EdlCreds = new S3({
-      //   apiVersion: '2006-03-01',
-      //   accessKeyId: credentials.AccessKeyId,
-      //   secretAccessKey: credentials.SecretAccessKey
-      // });
+      const payload = await callCumulusApi({
+        prefix: `${config.stackName}`,
+        payload: {
+          httpMethod: 'GET',
+          resource: '/{proxy+}',
+          path: '/s3credentials'
+        }
+      });
+      creds = JSON.parse(payload.body);
     });
 
-    it('the data can be downloaded from within the same region', () => {
+    it('the expected user can assume same region access', async () => {
+      const credentials = new Credentials(creds);
+      const sts = new STS({ credentials });
+      const whoami = await sts.getCallerIdentity().promise();
 
+      expect(whoami.Arn).toMatch(/arn:aws:sts::\d{12}:assumed-role\/s3-same-region-access-role\/userName.*/);
+      expect(whoami.UserId).toMatch(/.*:userName.*/);
     });
```

Comment:
Also to be clear, only this one test is currently being run, the rest should work after NGAP work.